scripts/postcss/css-url-to-variables.js

```diff
@@ -55,15 +55,16 @@ function addResolvedVariablesToRootSelector(root, { Rule, Declaration }) {
 
 function populateMapWithIcons(map, cssFileLocation) {
     const location = cssFileLocation.match(/(.+)\/.+\.css/)?.[1];
-    if (map.has(location)) {
+    const sharedObject = map.get(location);
+    if (sharedObject?.["icon"]) {
```

Comment:
Wondering what the icon property is for ... So the idea here is that you have one shared object for all the values that the postcss plugins need to output, and this plugin in particular writes to the `icon` property?

I do wonder if the complication of only setting icon once per file is worth it given we need to redo all the complicated work anyway (parsing the css) and the `map.set` should have a very minimal runtime cost. Or is this not just about performance and will something break if we set it multiple times (for each theme variant)? Anyway not a big deal, fine to leave it as it is.